test/test_gettab.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
```

Comment:
Comments please. What is this file, who wrote it, and what's it for?

Reply:
done.